Projecten3/EvaAndroid/app/src/main/java/projecten3_h6/evaandroid/Fragments/ChallengeFragment.java

```diff
@@ -0,0 +1,84 @@
+package projecten3_h6.evaandroid.Fragments;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import projecten3_h6.evaandroid.Adapters.AchievementDetailAdapter;
+import projecten3_h6.evaandroid.Adapters.ChallengeAdapter;
+import projecten3_h6.evaandroid.Domain.Achievement;
+import projecten3_h6.evaandroid.Domain.Challenge;
+import projecten3_h6.evaandroid.Domain.Day;
+import projecten3_h6.evaandroid.Domain.EvaApplication;
+import projecten3_h6.evaandroid.Domain.User;
+import projecten3_h6.evaandroid.R;
+
+/**
+ * Created by jensleirens on 25/07/2017.
+ */
+
+public class ChallengeFragment extends Fragment {
+
+    @BindView(R.id.challengeRecyclerView)RecyclerView mRecycler;
+
+    List<Challenge> challenges = new ArrayList<>();
+    protected RecyclerView.LayoutManager mLayoutManager;
+    private ChallengeAdapter adapter;
+    public static EvaApplication app;
```

Comment:
Avoid using too many static variables. If you need this in your adapter, pass it on through the constructor.